indico/modules/events/management/controllers.py

```diff
@@ -178,3 +178,13 @@ def _update_session_coordinator_privs(self, form):
             log_msg = 'Session coordinator privilege changed to {}: {}'.format(form.data[priv_field],
                                                                                COORDINATOR_PRIV_TITLES[priv_field])
             self.event_new.log(EventLogRealm.management, EventLogKind.positive, 'Protection', log_msg, session.user)
+
+
+class RHMoveEvent(RHConferenceModifBase):
```

Comment:
I guess this is in this file because we're gonna use it from the event too? But, in any case, wouldn't it make more sense to have it as a part of `controllers.management`? After all, we're talking of a category-level operation, right (I mean, there is a target category)?